sql/core/src/test/resources/sql-tests/inputs/transform.sql

```diff
@@ -183,3 +189,114 @@ SELECT a, b, decode(c, 'UTF-8'), d, e, f, g, h, i, j, k, l FROM (
     NULL DEFINED AS 'NULL'
   FROM t
 ) tmp;
+
+SELECT TRANSFORM (b, a, CAST(c AS STRING))
```

Comment:
nit: follow the other formats: `TRANSFORM (` -> `TRANSFORM(`

Reply:
Done